src/components/Modal/StakingChecklistModal/index.tsx

```diff
@@ -28,29 +31,29 @@ const StakingChecklistModal: FC<BaseModalProps & { stakeAmount: string }> = ({
   return (
     <>
       <ModalHeader display="flex" alignItems="baseline">
-        <H5 mr={2}>Stake Tokens</H5>
-        <BodySm>(Step 1)</BodySm>
+        <H5 mr={2}>Staking timeline</H5>
       </ModalHeader>
-      <ModalCloseButton />
-      <ModalBody>
-        <Stack spacing={6}>
-          <InfoBox variant="modal">
-            <BodyLg color={useColorModeValue("gray.700", "white")} as="span">
-              <H5 mb={4}>Staking in Threshold requires running a node</H5>
-              <BodyLg>
-                Please review the staking timeline below for a handy overview of
-                the staking requirements.
-              </BodyLg>
-            </BodyLg>
-          </InfoBox>
-          <Divider />
-          <StakingTimeline />
-          <StakingContractLearnMore mt="4rem !important" />
-          <Divider />
-        </Stack>
+      <ModalCloseButton top="3" />
+      <ModalBody py="0">
+        <InfoBox variant="modal">
+          <BodyLg color={useColorModeValue("gray.700", "white")} as="span">
+            <H5 as="h4">
+              Review the timeline carefully for an overview of the requirements.
+            </H5>
+          </BodyLg>
+        </InfoBox>
+        <StakingTimeline mt="4" statuses={[FlowStepStatus.inactive]} />
+        <Alert status="warning" mt="6">
+          <AlertIcon />
+          <AlertDescription>
```

Comment:
Let's add proper color to the alert description:

```suggestion
          <AlertDescription color={useColorModeValue("gray.700", "white")}>
```

Reply:
Applied 🙇🏼 
Ref commit: cebf705a7f63a13d3d1a0e5dfdbcc543390fc6a8